javascript-light-bulb/main.js

```diff
@@ -0,0 +1,19 @@
+
+var $body = document.querySelector('.light');
+var $container = document.querySelector('.switch-container-light');
+var $switch = document.querySelector('.switch-light');
+
+var isOff = true;
+
+$switch.addEventListener('click', function (event) {
+  if (isOff) {
+    $container.className = 'switch-container-dark';
+    $switch.className = 'switch-dark';
+    $body.className = 'dark';
```

Comment:
So we are turning off the light if it `isOff`? That sounds backwards. You should either rename your variable to `isOn` or reverse your conditional blocks here.